tests/io/v3/vm/test_folders.py

```diff
@@ -0,0 +1,67 @@
+import responses
+
+VM_FOLDERS_BASE_URL = 'https://cloud.tenable.com/api/v3/folders'
+SAMPLE_FOLDER_ID = 18
+SAMPLE_FOLDER = {
+    'unread_count': 0,
+    'custom': 0,
+    'default_tag': 0,
+    'type': 'main',
+    'name': 'my scans',
+    'id': SAMPLE_FOLDER_ID
+}
+
+
+@responses.activate
+def test_create(api):
+    '''
+    Test vm folders create method
+    '''
+    responses.add(
+        responses.POST,
+        VM_FOLDERS_BASE_URL,
+        json=SAMPLE_FOLDER
```

Comment:
it only returns ID as a response. pls update the json format

Reply:
Updated

Reply:
@parvdesai-tenable it shouldn't be like this == SAMPLE_FOLDER = {
    'unread_count': 0,
    'custom': 0,
    'default_tag': 0,
    'type': 'main',
    'name': 'my scans',
    'id': SAMPLE_FOLDER_ID
},

Instead it should be like -- 
SAMPLE_FOLDER = {
    'id': SAMPLE_FOLDER_ID
}

Reply:
We used the same SAMPLE_FOLDER for list folders test. So in order to cater the need of that test too i did this.